src/talks_tweeter_sup.erl

```diff
@@ -48,8 +48,11 @@ init([]) ->
                        type => worker,
                        modules => [tt_importer]
                       },
+
+    ActionInterval = {0, 0, 1},
+    TimeWindow = {0, 0, 3},
     SchedulerWorker = #{id => tt_scheduler,
-                        start => {tt_scheduler, start_link, []},
+                        start => {tt_scheduler, start_link, [ActionInterval, TimeWindow]},
```

Comment:
ActionInterval is time between tt_scheduler asking tt_store for talks between (TimeBeforeTalk; TimeBeforeTalk + TimeWindow).